src/app/controllers/censusdata/create.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name flightNodeApp.controller:CensusDataCreateController
+ * @description
+ * # CensusDataCreateController
+ * Controller for the create census form.
+ */
+angular.module('flightNodeApp')
+    .controller('CensusDataCreateController', 
+    ['$scope',
+    function ($scope) {
+		$scope.loading = true;
+      
+        //TODO: Get all this data from DB through API calls. Since it is all lookup data, we may cache it through out the application's lifetime.
+        $scope.tideInfo = [
+            { Id: 1, Description: "Water level appears high" },
+            { Id: 2, Description: "Water level appears low" },
+            { Id: 3, Description: "Wind driven" },
+            { Id: 4, Description: "Non-tidal" },      
+        ];
```

Comment:
One thing I've done in other screens is put this sort of lookup data into another container ("data), instead of directly putting it inside `$scope`. Thus it would be `$scope.data.tideInfo = ...`